### DISCUSSION TITLE
Hodnocení akcí účastníky

### DISCUSSION THREAD

**Discussion by Grejdy** [bosancz/bosan.cz#581] [Ideas]
Nevím ještě, jak přesně to pojmout (jestli prostě jako nějaký oddíl ve veřejné sekci nebo jestli to dát do interní a zařídit to tak, aby mohli mít profily v interní i třeba děti) ale přišlo by mi zajímavý, kdyby byla možnost u každé proběhlé akce ohodnotit ubytování, program, lokalitu… z různých pozic (vedoucí akce, vedoucí, dítě, …) Vedoucí by tím dostal nějaký report na to, jak se mu akce povedla, co by mohl zlepšit a zároveň by i ostatní vedoucí mohli jednoduše zjistit třeba která chata je na podzimky dobrá a kam nejezdit. Také by se dal třeba oceňovat vedoucí nejlepších akcí atp.

**Comment by jhavelik** [comment]
@Grejdy opět spíš do discussions, BTW masivní featura na kterou nebude mít nikdo čas a patří spíše do repa k interní sekci než veřejné.
Pokud má někdo zájem o feedback tak ho může zkusit získat od dětí jinak než přes web (např. GForm), šel by asi připravit nějaký template který si vedoucí upraví podle svého.

Názory? @SmallhillCZ @DavidTvrdy @elenfant

**Reply by Grejdy** [reply]
To právě třeba u tří jezů už pár let dělám… proto mě napadlo, že by bylo příjemné mít i lepší možnosti, než jen připíchnout qr-kód na nástěnku.
Ale rozumím, že je to poměrně velká věc, ke které se minimálně dlouho nikdo nedostane.

**Reply by DavidTvrdy** [reply]
@Grejdy Jaké máš zatím zkušenosti s hodnocením 3J? Dostáváš od dětí nějaké odpovědi? Jsou relevantní?

Nevím, přijde mi, že ptaním se dětí na místě třeba po hře nebo i po akci, jak se jim co líbilo, dostane člověk podobně kvalitní zpětnou vazbu. Určitě ne o tolik horší, aby se nám to teď vyplatilo implementovat na web. GForm template už 5 let máme na sdíleném disku "Akce", nevím, jestli ho někdo někdy zkusil použít.

**Reply by Grejdy** [reply]
Zatím jsem si ho letos úplně nevyhodnocoval, ale když jsem do něj sem tam nakoukl, tak tam byly i celkem zajímavý informace. Ale je pravda, že to jsou vesměs informace spíš od vedoucích, než od dětí.

Se soubory kdesi v hloubi g-drivu je věčný problém, že 95% vedoucích o nich většinou ani neví.:-)

**Reply by Goliash2** [reply]
Myslím si, že je dobrý mít někde archivované reakce z minula, zejména u akcí, které se opakují. Řekněme, že bude příští rok 3J organizovat někdo jiný a Grejdy nebude úplně po ruce. V takovém případě bude dobré pro nového vedoucího akce se podívat na loňské zhodnocení + by případně mohl hodnocení/zkušenosti napsat i sám vedoucí akce. Díky tomu pak může být akce lepší a dá se vyvarovat některým chybám, příp. využít uvedené zkušenosti, kontakty. Ptaní se dětí je super, ale za rok už nebudeš vědět, co ti k akci před rokem řekli. U Šánu pak ze zkušeností vznikají chytristiky s tipy jak se konkrétní úlohy dobře zhostit, ale i tam mi vždycky chybělo, že se nikde elektronicky neukládaly reakce třeba na CTH (takové ty komentáře typu: "bylo moc stanovišťovek", "tahle hra se mi líbila protože...", "v této hře byl super tenhle element/nápad" atd.). Tohle by se mělo ideálně někam zapisovat už v průběhu tábora.

**Comment by Grejdy** [comment]
Tohle umí i diskuze?! Hustý!
